Package.swift

```diff
@@ -84,14 +96,18 @@ let package = Package(
       url: "https://github.com/paulb777/nanopb.git",
       .revision("849a9b82233aaa5af4286c6c594cd831ca48651c")
     ),
+    .package(
+      name: "abseil",
+      url: "https://github.com/paulb777/abseil-cpp.git",
```

Comment:
For the purposes of *internal* experimentation this seems reasonable, but I don't think we should publish this to end users until we've settled how we're going to actually make these dependencies work. The outcome I don't want is that we end up stuck maintaining these ourselves because we've conditioned enough users to use Firestore via SPM that we can't call the experiment a failure.

Reply:
Comment aside:  I wouldn't be surprised to see SPM establish itself as a C++ dependecy manager, even if it currently lacks features vs CMake. Looking forward to see how Windows support in 5.3 will push that. I also would love to see Android NDK support Swift/SPM..